client/src/components/+account/summary.tsx

```diff
@@ -47,7 +47,7 @@ export function Summary({ published, bookmarks, downloaded, currentSubscription
               Your Subscription
             </UiText>
             {
-              currentSubscription ? {currentSubscription} : "No Subscription"
+              currentSubscription?currentSubscription: "No Subscription"
```

Comment:
More simple like this
```suggestion
    currentSubscription || "No Subscription"
```

Reply:
done